subsys/usb/usb_c/usbc_pe_snk_states.c

```diff
@@ -15,6 +15,27 @@ LOG_MODULE_DECLARE(usbc_stack, CONFIG_USBC_STACK_LOG_LEVEL);
 #include "usbc_pe_common_internal.h"
 #include "usbc_stack.h"
 
+/**
+ * @brief Initialize the Source Policy Engine layer
+ */
+void pe_snk_init(const struct device *dev)
+{
+	struct usbc_port_data *data = dev->data;
+	struct policy_engine *pe = data->pe;
+
+	/* Initial role of sink is UFP */
+	pe_set_data_role(dev, TC_ROLE_UFP);
+
+	/* Initialize timers */
+	usbc_timer_init(&pe->pd_t_typec_sink_wait_cap, PD_T_TYPEC_SINK_WAIT_CAP_MAX_MS);
+	usbc_timer_init(&pe->pd_t_ps_transition, PD_T_SPR_PS_TRANSITION_NOM_MS);
+	usbc_timer_init(&pe->pd_t_chunking_not_supported, PD_T_CHUNKING_NOT_SUPPORTED_NOM_MS);
```

Comment:
`ChunkingNotSupportedTimer` doesn't seem sink-specific to me.

Reply:
Moved to common code.